content/en/_index.html

```diff
@@ -43,7 +43,7 @@ <h4>Run Anywhere</h4>
 <section id="video">
     <div class="light-text">
         <h2>Kubernetes: Finally&hellip; A True Cloud Platform</h2>
-        <p>Sam Ghods, Co-Founder and Services Architect at Box, gives a passionate talk last year at KubeCon Seattle showing that with Kubernetes, we have for first time a universal interface that one can build real deployment tooling against.</p>
+        <p>Sam Ghods, Co-Founder and Services Architect at Box, gives a passionate talk last year at KubeCon Seattle showing that with Kubernetes, we have for the first time a universal interface that one can build real deployment tooling against.</p>
```

Comment:
1. arguably job titles after the name need to capitalization (e.g. engineer)
2. `last year` is no longer valid as this talk is from 2016. better state the year.

```
Sam Ghods, co-founder and services architect at Box, gives a passionate talk at KubeCon 2016 Seattle showing that with Kubernetes, we have for the first time a universal interface that one can build real deployment tooling against.
```